[PATCH] assume storing is success if not sound card device

Upstream-Statue: Pending

Systemd will report failure when run alsa-*, if the machine has not the
sound card. To void this annoyed message, alsa-restore/alsa-state ignore
all the exit codes by prefixing "-" in ExecStart, like:

   alsa-utils-1.0.29$ grep "=-" ./ -r|grep service.in
   ./alsactl/alsa-restore.service.in:ExecStart=-@sbindir@/alsactl restore
   ./alsactl/alsa-state.service.in:ExecStart=-@sbindir@/alsactl -s -n 19 -c rdaemon
   ./alsactl/alsa-state.service.in:ExecStop=-@sbindir@/alsactl -s kill save_and_quit
   lsa-utils-1.0.29$

But alsa-store.service.in is missing, and better solution is to ignore
the exit code 19 which means not sound card device, not all exit code

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 alsactl/alsa-store.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/alsactl/alsa-store.service.in b/alsactl/alsa-store.service.in
index f1a56bb..68ca529 100644
--- a/alsactl/alsa-store.service.in
+++ b/alsactl/alsa-store.service.in
@@ -13,3 +13,4 @@ Before=shutdown.target
 Type=oneshot
 ExecStart=@sbindir@/alsactl store
 StandardOutput=syslog
+SuccessExitStatus=0 19
-- 
1.9.1

